.github/workflows/windows.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         mkdir build
         mkdir c:/project
         cd build
-        cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=c:/project -DDEAL_II_WITH_ZLIB=off -DDEAL_II_CXX_FLAGS="-WX" -T host=x64 -A x64 ..
+        cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=c:/project -DDEAL_II_WITH_ZLIB=off -DDEAL_II_CXX_FLAGS="-WX /D FE_EVAL_FACTORY_DEGREE_MAX=2" -T host=x64 -A x64 ..
```

Comment:
Is it so simple to define a C macro? It it wouldn't work just override manually the value of

https://github.com/dealii/dealii/blob/0a36820bd837e3d391a5dd9551be860709db08ed/include/deal.II/matrix_free/evaluation_template_factory_internal.h#L24

... and we find a solution after we have seen what the benefits are!

Reply:
OK. It worked (now I see that you are directly modify the CXX flags)! I'll take another look at the code later today!

Reply:
It was Martin, just to be clear! I shouldn't take credit here :) 

I'll prepare the PR of the Piola for the affine case right away.

Reply:
@kronbichler Why aren't we do it like this in ExaDG? We also have somewhere the possibility if fast evaluation is enabled or not...

Reply:
One could do that as well - the reason why I opted against it up to now was that deal.II compiles considerably more code than ExaDG does, as it repeats the instantiations also for narrower SIMD (ExaDG picks only the widest version), and that makes deal.II libraries considerably fatter. The other question is whether ExaDG really needs all instantiations up to 15 with the fast path, or whether 7 or 8 would suffice and then use the slower path afterwards.